ArteryMessageLoggingSpec: Message logging must not be on if debug logging not enabled

https://jenkins.akka.io:8498/job/akka-artery-tcp-remote-tests/38/consoleFull

'Connection refused' seems suspicious here?

```
[info] ArteryMessageLoggingSpec:
Coroner Thread Count starts at 43 in akka.remote.ArteryMessageLoggingSpec
[info] Message logging
[WARN] [03/28/2018 10:04:32.911] [MessageLoggingSpec-akka.remote.default-remote-dispatcher-7] [akka.stream.Log(akka://MessageLoggingSpec/system/StreamSupervisor-29)] [outbound connection to [akka://remote-sys@localhost:49825], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(localhost:49825,None,List(),Some(5000 milliseconds),true)] failed because of Connection refused
[ERROR] [03/28/2018 10:04:32.911] [MessageLoggingSpec-akka.remote.default-remote-dispatcher-7] [RestartWithBackoffFlow(akka://MessageLoggingSpec)] Restarting graph due to failure (akka.stream.StreamTcpException: Tcp command [Connect(localhost:49825,None,List(),Some(5000 milliseconds),true)] failed because of Connection refused)
[WARN] [03/28/2018 10:04:32.912] [MessageLoggingSpec-akka.remote.default-remote-dispatcher-7] [akka.stream.Log(akka://MessageLoggingSpec/system/StreamSupervisor-29)] [outbound connection to [akka://remote-sys@localhost:49825], control stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(localhost:49825,None,List(),Some(5000 milliseconds),true)] failed because of Connection refused
[ERROR] [03/28/2018 10:04:32.912] [MessageLoggingSpec-akka.remote.default-remote-dispatcher-7] [RestartWithBackoffFlow(akka://MessageLoggingSpec)] Restarting graph due to failure (akka.stream.StreamTcpException: Tcp command [Connect(localhost:49825,None,List(),Some(5000 milliseconds),true)] failed because of Connection refused)
[WARN] [03/28/2018 10:04:34.918] [MessageLoggingSpec-akka.remote.default-remote-dispatcher-14] [akka.stream.Log(akka://MessageLoggingSpec/system/StreamSupervisor-29)] [outbound connection to [akka://remote-sys@localhost:49825], control stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(localhost:49825,None,List(),Some(5000 milliseconds),true)] failed because of Connection refused
[WARN] [03/28/2018 10:04:34.918] [MessageLoggingSpec-akka.remote.default-remote-dispatcher-10] [akka.stream.Log(akka://MessageLoggingSpec/system/StreamSupervisor-29)] [outbound connection to [akka://remote-sys@localhost:49825], message stream] Upstream failed, cause: StreamTcpException: Tcp command [Connect(localhost:49825,None,List(),Some(5000 milliseconds),true)] failed because of Connection refused
[ERROR] [03/28/2018 10:04:34.918] [MessageLoggingSpec-akka.remote.default-remote-dispatcher-10] [RestartWithBackoffFlow(akka://MessageLoggingSpec)] Restarting graph due to failure (akka.stream.StreamTcpException: Tcp command [Connect(localhost:49825,None,List(),Some(5000 milliseconds),true)] failed because of Connection refused)
[ERROR] [03/28/2018 10:04:34.918] [MessageLoggingSpec-akka.remote.default-remote-dispatcher-14] [RestartWithBackoffFlow(akka://MessageLoggingSpec)] Restarting graph due to failure (akka.stream.StreamTcpException: Tcp command [Connect(localhost:49825,None,List(),Some(5000 milliseconds),true)] failed because of Connection refused)
[info] - must not be on if debug logging not enabled *** FAILED *** (3 seconds, 10 milliseconds)
[info]   java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsgClass waiting for class akka.actor.ActorIdentity
[info]   at scala.Predef$.assert(Predef.scala:170)
[info]   at akka.testkit.TestKitBase$class.expectMsgClass_internal(TestKit.scala:509)
[info]   at akka.testkit.TestKitBase$class.expectMsgType(TestKit.scala:482)
[info]   at akka.testkit.TestKit.expectMsgType(TestKit.scala:896)
[info]   at akka.remote.MessageLoggingSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply(MessageLoggingSpec.scala:62)
[info]   at akka.remote.MessageLoggingSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply(MessageLoggingSpec.scala:58)
[info]   at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]   at org.scalatest.WordSpecLike$$anon$1.apply(WordSpecLike.scala:1078)
[info]   at org.scalatest.TestSuite$class.withFixture(TestSuite.scala:196)
[info]   at akka.testkit.AkkaSpec.withFixture(AkkaSpec.scala:60)
[info]   at org.scalatest.WordSpecLike$class.invokeWithFixture$1(WordSpecLike.scala:1075)
[info]   at org.scalatest.WordSpecLike$$anonfun$runTest$1.apply(WordSpecLike.scala:1088)
[info]   at org.scalatest.WordSpecLike$$anonfun$runTest$1.apply(WordSpecLike.scala:1088)
[info]   at org.scalatest.SuperEngine.runTestImpl(Engine.scala:289)
[info]   at org.scalatest.WordSpecLike$class.runTest(WordSpecLike.scala:1088)
[info]   at akka.testkit.AkkaSpec.runTest(AkkaSpec.scala:60)
[info]   at org.scalatest.WordSpecLike$$anonfun$runTests$1.apply(WordSpecLike.scala:1147)
[info]   at org.scalatest.WordSpecLike$$anonfun$runTests$1.apply(WordSpecLike.scala:1147)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:396)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:384)
[info]   at scala.collection.immutable.List.foreach(List.scala:392)
[info]   at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:384)
[info]   at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:373)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:410)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:384)
[info]   at scala.collection.immutable.List.foreach(List.scala:392)
[info]   at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:384)
[info]   at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:379)
[info]   at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:461)
[info]   at org.scalatest.WordSpecLike$class.runTests(WordSpecLike.scala:1147)
[info]   at akka.testkit.AkkaSpec.runTests(AkkaSpec.scala:60)
[info]   at org.scalatest.Suite$class.run(Suite.scala:1147)
[info]   at akka.testkit.AkkaSpec.org$scalatest$WordSpecLike$$super$run(AkkaSpec.scala:60)
[info]   at org.scalatest.WordSpecLike$$anonfun$run$1.apply(WordSpecLike.scala:1192)
[info]   at org.scalatest.WordSpecLike$$anonfun$run$1.apply(WordSpecLike.scala:1192)
[info]   at org.scalatest.SuperEngine.runImpl(Engine.scala:521)
[info]   at org.scalatest.WordSpecLike$class.run(WordSpecLike.scala:1192)
[info]   at akka.testkit.AkkaSpec.org$scalatest$BeforeAndAfterAll$$super$run(AkkaSpec.scala:60)
[info]   at org.scalatest.BeforeAndAfterAll$class.liftedTree1$1(BeforeAndAfterAll.scala:213)
[info]   at org.scalatest.BeforeAndAfterAll$class.run(BeforeAndAfterAll.scala:210)
[info]   at akka.testkit.AkkaSpec.run(AkkaSpec.scala:60)
[info]   at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:314)
[info]   at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:480)
[info]   at sbt.ForkMain$Run$2.call(ForkMain.java:301)
[info]   at sbt.ForkMain$Run$2.call(ForkMain.java:291)
[info]   at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info]   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[info]   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[info]   at java.lang.Thread.run(Thread.java:745)
```

--- a/akka-remote/src/test/scala/akka/remote/MessageLoggingSpec.scala
+++ b/akka-remote/src/test/scala/akka/remote/MessageLoggingSpec.scala
@@ -52,8 +52,9 @@ class ClassicMessageLoggingSpec extends MessageLoggingSpec(config(false))
 
 abstract class MessageLoggingSpec(config: Config) extends AkkaSpec(config) with ImplicitSender {
 
-  val remoteSystem = ActorSystem("remote-sys", config)
+  val remoteSystem = ActorSystem("remote-sys", ConfigFactory.load(config))
   val remoteAddress = remoteSystem.asInstanceOf[ExtendedActorSystem].provider.getDefaultAddress
+
   "Message logging" must {
     "not be on if debug logging not enabled" in {
       remoteSystem.actorOf(Props[BadActor], "bad")

